mods/webui/src/theme.ts

```diff
@@ -29,6 +29,16 @@ const fnLight = createTheme({
       "800": "#053204",
       "900": "#011900"
     },
```

Comment:
Added secondary colors to the palette 
using collors from the brand identity grayscale
![Screenshot 2024-12-11 at 9 35 41 AM](https://github.com/user-attachments/assets/19e42189-40c4-4dee-84b3-4379e7bd6ce5)